Source/Serilog.Enrichers.Span/ActivityEnricher.cs

```diff
@@ -9,6 +9,60 @@ namespace Serilog.Enrichers.Span
     /// </summary>
     public class ActivityEnricher : ILogEventEnricher
     {
+#if NET5_0
+        /// <summary>
+        /// Enrich the log event.
+        /// </summary>
+        /// <param name="logEvent">The log event to enrich.</param>
+        /// <param name="propertyFactory">Factory for creating new properties to add to the event.</param>
+        public void Enrich(LogEvent logEvent, ILogEventPropertyFactory propertyFactory)
+        {
+            var activity = Activity.Current;
+
+            if (activity is not null)
+            {
+                AddSpanId(logEvent, activity);
+                AddTraceId(logEvent, activity);
+                AddParentId(logEvent, activity);
+            }
+        }
+
+        private static void AddSpanId(LogEvent logEvent, Activity activity)
+        {
+            var property = activity.GetCustomProperty("Serilog.SpanId");
+            if (property is null || property is not LogEventProperty logEventProperty)
+            {
+                logEventProperty = new LogEventProperty("SpanId", new ScalarValue(activity.GetSpanId()));
```

Comment:
If we care about allocations, we should use constants for the strings in this class.

Reply:
Done!